recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "spatialpandas" %}
-{% set version = "0.4.9" %}
+{% set version = "0.4.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 666c7b33fe0e8516b88029beaf23f7260375ec42997e5a2e40b3e689d73f28cf
+  sha256: 032e24ebb40f75c5c79cb79d7c281f2990e69ba382c0b24acb53da7bba60851c
 
 build:
   number: 0
```

Comment:
`skip: True  #[py<39 or s390x]`
see https://github.com/holoviz/spatialpandas/blob/v0.4.10/setup.py#L56

Reply:
Is there missing `pyct` in `host`? see https://github.com/holoviz/spatialpandas/blob/b5da5119a41a200c9a0209b472838c75b0ff81e1/pyproject.toml#L4

Reply:
I think you can safely replace `param 1.13.0` by `param`

Reply:
> Instead I decided to update the host dependencies so they match the build requirements as pinned in pyproject.toml. Let me know what you think of this approach.

Ignore my comment, the linter complained, I removed all the pins in the host section.